src/UNL/MediaHub/Manager/PostHandler.php

```diff
@@ -515,6 +550,133 @@ function handleDeleteMedia()
 
         UNL_MediaHub::redirect(UNL_MediaHub_Manager::getURL());
     }
+    
+    function handleAmara()
```

Comment:
Let's get in the habit of updating files to be more PSR4 compliant, but adding the missing visibility keyword (`public`).